### ISSUE TITLE
Switching phones, how to transfer data?

### ISSUE THREAD

**Issue by cessen** [federicoiosue/Omni-Notes#825]
I've been using Omni Notes for quite some time, and have accrued quite a few notes.  The last time I changed phones, I didn't have so many notes, so I just manually re-entered them.  But this time that would be a bit painful.

I searched for related issues, and found #473 and a couple other similar ones.  In these issues people are using the backup system to transfer the notes, which is also what I first tried.  However, connecting my phone to my computer and browsing the phone's contents, I cannot find any `Omni Notes` directory on my phone to copy.  I also tried a command-line search with regex, and it also turned up zero files with either `omni` or `Omni` in the name.

It basically seems like Omni Notes somehow has zero visible presence on my phone, which is really weird.

Any help would be appreciated!

**Comment by federicoiosue** [comment]
Hi Nathan, which version of the app are you using?
What if you try to go back to settings (once the backup has been created) anche check into the restore option? Some backup is shown there?

**Comment by cessen** [comment]
Hi Federico,

I'm on Omni Notes version 6.0.5 (which I think is the latest?).  I can see the backups in-app just fine in the restore option, yes.

But I can't find anything on the phone relating to Omni Notes at all--much less the backups--when browsing its filesystem via USB file-transfer mode on my computer.  I also tried giving one of the backups a custom name, and doing an automated search for that on the filesystem, but no luck.

This phone is on an older version of Android.  Maybe that's part of the problem?  But it still seems strange that I can't find anything at all related to Omni Notes on the phone's filesystem.

**Comment by federicoiosue** [comment]
Nope, it should cause no issue at all. Moreover, if you're able to see the backup you've done previously into the list of restorable ones the folder exists for sure.

It must be into the internal sdcard root directory. Maybe your device supports external storage and you're looking inside that one? 
I think root is needed to check device's logs, but if it's not (or you're root) you could check logs to verify the full backups path!

**Comment by cessen** [comment]
Hmm, I don't have a rooted phone.  So is that the problem?  And if so, is there a way to do this that doesn't involve rooting my phone?

Btw, I really appreciate you taking the time to help!

**Comment by federicoiosue** [comment]
No, it's not a problem, was just a usefull thing to have to be sure to check internal storage content.

I really don't know why you can't find that directory, the following is the output of a directory listing of my OnePlus 5T device (I actually keep installed both the stable and the alpha version of the app):
```
dumpling:/ $ ls -al storage/emulated/0/ | grep Omni
drwxrwx---  3 root everybody  4096 2021-05-07 10:28 Omni Notes
drwxrwx---  7 root everybody  4096 2021-05-09 11:27 Omni Notes Alpha
```

If you thing this could help send me by private message or email (support@omninotes.app) a screenshot or whatever else of your storage

**Comment by cessen** [comment]
Huh, so this weird: I plugged my phone into my computer again to run the check for you, and now the `Omni Notes` directory is there.  I'm 90% it wasn't there before, as I ran a full recursive search before, similar to your command above but using `find` instead of `ls`:

```
find | grep [Oo]mni
```

I suppose it's always possible there was user error here, but I don't think so as it's a pretty brain-dead thing to check, and it is indeed right in the root of the storage where I was looking for it before.

In any case, problem solved.  I do wonder why it wasn't showing up before, though.  My guess is something funky with my phone, rather than with Omni Notes.
